src/test/regress/expected/multi_extension.out

```diff
@@ -1014,6 +1014,10 @@ SELECT * FROM multi_extension.print_extension_changes();
                                                                                                                       | function citus_shards_on_worker() SETOF record
                                                                                                                       | function citus_stat_activity() SETOF record
                                                                                                                       | function create_distributed_function(regprocedure,text,text,boolean) void
+                                                                                                                      | function get_nodename(integer) text
```

Comment:
We are introducing `get_nodeid_for_groupid ` in #5766. So, maybe aling with that?

maybe: `get_nodename_for_nodeid`,  `get_nodeport_for_nodeid`, `get_nodeid_for_gpid`, `get_localpid_for_gpid`

Reply:
thinking out loud: alternative could be to stick to the citus_ prefix (e.g. citus_nodename_for_nodeid), similar to citus_backend_gpid.